### ISSUE TITLE
Use structured logging syntax with ILogger

### ISSUE THREAD

**Issue by martincostello** [justeat/JustSaying#415]
I noticed that the `SqsNotificationListener` class uses `ILogger` from .NET Standard, but using string interpolation rather than structured logging syntax, [for example](https://github.com/justeat/JustSaying/blob/0739fdf4d90d4e75d55a762ac4a029a434786de7/JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs#L110):

```csharp
log.LogInformation($"[{task.Status}] Stopped Listening - {queueInfo}");
```

We should review the logging messages throughout the codebase and prefer structured-logging syntax when data is formatted into messages similar to the below example so that if a logging implementation such as Serilog is in use, logs can be filtered/searched by the `MessageTemplate` property as well as the `Message` property.

```csharp
log.LogInformation("[{TaskStatus}] Stopped Listening - {QueueInfo}", task.Status, queueInfo);
```

**Comment by AnthonySteele** [comment]
If the Standard logger interface supports this syntax, then this would be great!

**Comment by martincostello** [comment]
It's up to the actual logging implementation. Serilog and Application Insights do for instance, others might just end up formatting it internally anyway and not ever generate a `MessageTemplate`, in which case the end result is the same in the log as the way the code is now.

**Comment by AnthonySteele** [comment]
I believe that NLog 4.5 (plain NLog without NLog.StructuredLogging) also [does structured logging in some form](https://github.com/NLog/NLog/wiki/How-to-use-structured-logging). It would be good to test it out by doing this.

**Comment by AnthonySteele** [comment]
I think that this is fixed by #493 .

IMHO it needs beta testing with load and an ELK stack to be sure that everything is of production quality, but we're not ready for that just yet.

Close this issue now?

**Comment by AnthonySteele** [comment]
They are all done, as far as I can see (and I checked)
